languages/rust/reference/merge-strategy.md

```diff
@@ -0,0 +1,34 @@
+# merging strategy
+
+The Rust track maintainers [have decided](https://github.com/exercism/v3/discussions/1725#discussion-7438) to write down our merging strategy to communicate expectations and encourage quality contributions.
+
+## general principles
+
+- we strive to accept _all_ contributions
+  - We'll bring contributions to a high standard of quality through collaborative, iterative review.
+  - We adopt a pessimistic merging approach. Think of maintainer reviews like the Rust compiler's feedback. The amount of feedback can feel overwhelming and frustrating. But we'll work with you to get it over the finish line! So keep up the good work.
+- One other maintainer should review a contributor's pull request.
+- At least one other maintainer should approve a pull request before merging it into the development branch.
```

Comment:
I agree with these, but the wording is awkward. In particular, the word "other" feels out of place.

Also, this is enforced by github. I guess the point is to encode as policy what's currently enforced?

```suggestion
- Every pull request must have at least 1 "Approve", and no "Request Changes", from a track maintainer before it can be merged.
- No maintainer can approve their own PR.
```

Reply:
Good points.
Exactly. I believe it is helpful to declare how we use GitHub's features. I've worked on other OSS projects where the approvals were not used or used inconsistently.
You also qualified "track maintainer". I like that and feel like I should update the terms to make it explicit. In the Rust track we are talking about track maintainers, not just anyone in the Exercism organization.